core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -138,6 +143,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime, threadNamePr
   val correlationId: AtomicInteger = new AtomicInteger(0)
   val brokerMetaPropsFile = "meta.properties"
   val brokerMetadataCheckpoints = config.logDirs.map(logDir => (logDir, new BrokerMetadataCheckpoint(new File(logDir + File.separator +brokerMetaPropsFile)))).toMap
+  var clusterId: String = null
```

Comment:
So, I think it's fine not to change other fields to private, but we should make this one private. It's easy to provide an accessor via a `def`. The reason to make it private is that it should not be available for mutation outside this class. If we need an accessor, we can name it `clusterId` and rename the variable `_clusterId` or something like that.